### PR TITLE
Pr 1555

### PR DESCRIPTION
**Pull request by ashutosh-rath02**
**Description**

This PR fixes #

**Notes for Reviewers**
Resolved issues with PR #1555 

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@iArchitSharma look good?

**Comment by iArchitSharma** [comment]
@ashutosh-rath02 Can you please add a screenshot of your changes to help it move smoothly?

**Comment by ashutosh-rath02** [comment]
Before:
image

After:
image

@iArchitSharma Sorry for the delay.

**Comment by saurabh100ni** [comment]
Hi @ashutosh-rath02  ,
Thanks for your PR! I encourage you to share this in the Website Meeting tomorrow at 6:30 PM IST. Share your screen, and walk us through it. It's cool to take and share feedback. Check the [Doc](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit) here to include this PR.

You're welcome at the meeting

Check to [community calendar](https://layer5.io/community/calendar) for more details of meetings at Layers.

**Review by leecalcote — CHANGES_REQUESTED** [review]
@ashutosh-rath02 will you remove any changes (like whitespace changes) that are not pertinent to this issue?

File: _sass/blog.scss [hunk 1]
```diff
@@ -94,8 +94,8 @@
 
     .post-date {
       display: flex;
-      justify-self: flex-end;
-      align-self: flex-end;
+      justify-self: flex-start;
```
Comment (vishalvivekm) [inline _sass/blog.scss:97]:
```suggestion

```
No `justify-self` in flex. could be removed.

**Comment by vishalvivekm** [comment]
@ashutosh-rath02 
post with authors: 
Screen Shot 2024-01-15 at 15 13 41

**Review by vishalvivekm — CHANGES_REQUESTED** [review]
Let's get feedback from other members of the community on call, in today's websites call.
I'll be adding this as an agenda item @ashutosh-rath02

**Comment by sudhanshutech** [comment]
@ashutosh-rath02 

image
image
image

**Comment by sudhanshutech** [comment]
still working on it @ashutosh-rath02 ?

**Comment by ashutosh-rath02** [comment]
> still working on it @ashutosh-rath02 ?

Yes, will push commits by tomorrow 😊

**Comment by ashutosh-rath02** [comment]
@sudhanshutech i am unable to see the recent blog in my branch.

**Comment by ashutosh-rath02** [comment]
Closing this PR

**Pull request closed without merging**